## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 511567`
Commit: `b7611bcedbf5fb9958a3f9e32d2328c41c85a7e0`

---
### `x86_64-linux`
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>vllm (python313Packages.vllm)</li>
    <li>vllm.dist (python313Packages.vllm.dist)</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 183 packages built:</summary>
  <ul>
    <li>frigate</li>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ale-py.dist</li>
    <li>python313Packages.anndata</li>
    <li>python313Packages.anndata.dist</li>
    <li>python313Packages.array-api-compat</li>
    <li>python313Packages.array-api-compat.dist</li>
    <li>python313Packages.arviz</li>
    <li>python313Packages.arviz.dist</li>
    <li>python313Packages.augmax</li>
    <li>python313Packages.augmax.dist</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.bambi.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.brax.dist</li>
    <li>python313Packages.chex</li>
    <li>python313Packages.chex.dist</li>
    <li>python313Packages.cleanlab</li>
    <li>python313Packages.cleanlab.dist</li>
    <li>python313Packages.clu</li>
    <li>python313Packages.clu.dist</li>
    <li>python313Packages.corner</li>
    <li>python313Packages.corner.dist</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.deepface.dist</li>
    <li>python313Packages.distrax</li>
    <li>python313Packages.distrax.dist</li>
    <li>python313Packages.dm-haiku</li>
    <li>python313Packages.dm-haiku.dist</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.dm-sonnet.dist</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.edward.dist</li>
    <li>python313Packages.equinox</li>
    <li>python313Packages.equinox.dist</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.evosax.dist</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flax.dist</li>
    <li>python313Packages.flowjax</li>
    <li>python313Packages.flowjax.dist</li>
    <li>python313Packages.flowmc</li>
    <li>python313Packages.flowmc.dist</li>
    <li>python313Packages.funsor</li>
    <li>python313Packages.funsor.dist</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.gymnasium.dist</li>
    <li>python313Packages.jax</li>
    <li>python313Packages.jax-cuda12-pjrt</li>
    <li>python313Packages.jax-cuda12-plugin</li>
    <li>python313Packages.jax.dist</li>
    <li>python313Packages.jaxlib (python313Packages.jaxlib-bin)</li>
    <li>python313Packages.jaxlib.dist (python313Packages.jaxlib-bin.dist)</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.jmp</li>
    <li>python313Packages.jmp.dist</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.keras.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.lineax</li>
    <li>python313Packages.lineax.dist</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mtcnn.dist</li>
    <li>python313Packages.mujoco-mjx</li>
    <li>python313Packages.mujoco-mjx.dist</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.numpyro</li>
    <li>python313Packages.numpyro.dist</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.nutpie.dist</li>
    <li>python313Packages.optax</li>
    <li>python313Packages.optax.dist</li>
    <li>python313Packages.optax.testsout</li>
    <li>python313Packages.optimistix</li>
    <li>python313Packages.optimistix.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>python313Packages.orbax-export</li>
    <li>python313Packages.orbax-export.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.paramax</li>
    <li>python313Packages.paramax.dist</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pettingzoo.dist</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pymanopt.dist</li>
    <li>python313Packages.pymc</li>
    <li>python313Packages.pymc.dist</li>
    <li>python313Packages.pytensor</li>
    <li>python313Packages.pytensor.dist</li>
    <li>python313Packages.pyvers</li>
    <li>python313Packages.pyvers.dist</li>
    <li>python313Packages.retinaface</li>
    <li>python313Packages.retinaface.dist</li>
    <li>python313Packages.rlax</li>
    <li>python313Packages.rlax.dist</li>
    <li>python313Packages.scanpy</li>
    <li>python313Packages.scanpy.dist</li>
    <li>python313Packages.scikit-bio</li>
    <li>python313Packages.scikit-bio.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.stable-baselines3.dist</li>
    <li>python313Packages.tensordict</li>
    <li>python313Packages.tensordict.dist</li>
    <li>python313Packages.tensorflow-datasets</li>
    <li>python313Packages.tensorflow-datasets.dist</li>
    <li>python313Packages.tensorflow-probability</li>
    <li>python313Packages.tensorflow-probability.dist</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.tianshou.dist</li>
    <li>python313Packages.treescope</li>
    <li>python313Packages.treescope.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vqgan-jax</li>
    <li>python313Packages.vqgan-jax.dist</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.waymax.dist</li>
    <li>python314Packages.anndata</li>
    <li>python314Packages.anndata.dist</li>
    <li>python314Packages.array-api-compat</li>
    <li>python314Packages.array-api-compat.dist</li>
    <li>python314Packages.augmax</li>
    <li>python314Packages.augmax.dist</li>
    <li>python314Packages.blackjax</li>
    <li>python314Packages.blackjax.dist</li>
    <li>python314Packages.chex</li>
    <li>python314Packages.chex.dist</li>
    <li>python314Packages.dm-haiku</li>
    <li>python314Packages.dm-haiku.dist</li>
    <li>python314Packages.equinox</li>
    <li>python314Packages.equinox.dist</li>
    <li>python314Packages.flowmc</li>
    <li>python314Packages.flowmc.dist</li>
    <li>python314Packages.funsor</li>
    <li>python314Packages.funsor.dist</li>
    <li>python314Packages.jax</li>
    <li>python314Packages.jax-cuda12-pjrt</li>
    <li>python314Packages.jax-cuda12-plugin</li>
    <li>python314Packages.jax.dist</li>
    <li>python314Packages.jaxlib (python314Packages.jaxlib-bin)</li>
    <li>python314Packages.jaxlib.dist (python314Packages.jaxlib-bin.dist)</li>
    <li>python314Packages.jaxopt</li>
    <li>python314Packages.jaxopt.dist</li>
    <li>python314Packages.jmp</li>
    <li>python314Packages.jmp.dist</li>
    <li>python314Packages.lineax</li>
    <li>python314Packages.lineax.dist</li>
    <li>python314Packages.mujoco-mjx</li>
    <li>python314Packages.mujoco-mjx.dist</li>
    <li>python314Packages.nifty8</li>
    <li>python314Packages.nifty8.dist</li>
    <li>python314Packages.optax</li>
    <li>python314Packages.optax.dist</li>
    <li>python314Packages.optax.testsout</li>
    <li>python314Packages.optimistix</li>
    <li>python314Packages.optimistix.dist</li>
    <li>python314Packages.orbax-checkpoint</li>
    <li>python314Packages.orbax-checkpoint.dist</li>
    <li>python314Packages.paramax</li>
    <li>python314Packages.paramax.dist</li>
    <li>python314Packages.pyvers</li>
    <li>python314Packages.pyvers.dist</li>
    <li>python314Packages.scanpy</li>
    <li>python314Packages.scanpy.dist</li>
    <li>python314Packages.scikit-bio</li>
    <li>python314Packages.scikit-bio.dist</li>
    <li>python314Packages.tensordict</li>
    <li>python314Packages.tensordict.dist</li>
    <li>python314Packages.treescope</li>
    <li>python314Packages.treescope.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>vllm (python313Packages.vllm)</li>
    <li>vllm.dist (python313Packages.vllm.dist)</li>
    <li>python314Packages.lineax</li>
    <li>python314Packages.lineax.dist</li>
    <li>python314Packages.optimistix</li>
    <li>python314Packages.optimistix.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 169 packages built:</summary>
  <ul>
    <li>frigate</li>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ale-py.dist</li>
    <li>python313Packages.anndata</li>
    <li>python313Packages.anndata.dist</li>
    <li>python313Packages.array-api-compat</li>
    <li>python313Packages.array-api-compat.dist</li>
    <li>python313Packages.arviz</li>
    <li>python313Packages.arviz.dist</li>
    <li>python313Packages.augmax</li>
    <li>python313Packages.augmax.dist</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.bambi.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.brax.dist</li>
    <li>python313Packages.chex</li>
    <li>python313Packages.chex.dist</li>
    <li>python313Packages.cleanlab</li>
    <li>python313Packages.cleanlab.dist</li>
    <li>python313Packages.corner</li>
    <li>python313Packages.corner.dist</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.deepface.dist</li>
    <li>python313Packages.distrax</li>
    <li>python313Packages.distrax.dist</li>
    <li>python313Packages.dm-haiku</li>
    <li>python313Packages.dm-haiku.dist</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.edward.dist</li>
    <li>python313Packages.equinox</li>
    <li>python313Packages.equinox.dist</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.evosax.dist</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flax.dist</li>
    <li>python313Packages.flowjax</li>
    <li>python313Packages.flowjax.dist</li>
    <li>python313Packages.flowmc</li>
    <li>python313Packages.flowmc.dist</li>
    <li>python313Packages.funsor</li>
    <li>python313Packages.funsor.dist</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.gymnasium.dist</li>
    <li>python313Packages.jax</li>
    <li>python313Packages.jax-cuda12-pjrt</li>
    <li>python313Packages.jax-cuda12-plugin</li>
    <li>python313Packages.jax.dist</li>
    <li>python313Packages.jaxlib (python313Packages.jaxlib-bin)</li>
    <li>python313Packages.jaxlib.dist (python313Packages.jaxlib-bin.dist)</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.jmp</li>
    <li>python313Packages.jmp.dist</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.keras.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.lineax</li>
    <li>python313Packages.lineax.dist</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mtcnn.dist</li>
    <li>python313Packages.mujoco-mjx</li>
    <li>python313Packages.mujoco-mjx.dist</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.numpyro</li>
    <li>python313Packages.numpyro.dist</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.nutpie.dist</li>
    <li>python313Packages.optax</li>
    <li>python313Packages.optax.dist</li>
    <li>python313Packages.optax.testsout</li>
    <li>python313Packages.optimistix</li>
    <li>python313Packages.optimistix.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>python313Packages.orbax-export</li>
    <li>python313Packages.orbax-export.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.paramax</li>
    <li>python313Packages.paramax.dist</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pettingzoo.dist</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pymanopt.dist</li>
    <li>python313Packages.pymc</li>
    <li>python313Packages.pymc.dist</li>
    <li>python313Packages.pytensor</li>
    <li>python313Packages.pytensor.dist</li>
    <li>python313Packages.pyvers</li>
    <li>python313Packages.pyvers.dist</li>
    <li>python313Packages.retinaface</li>
    <li>python313Packages.retinaface.dist</li>
    <li>python313Packages.rlax</li>
    <li>python313Packages.rlax.dist</li>
    <li>python313Packages.scikit-bio</li>
    <li>python313Packages.scikit-bio.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.stable-baselines3.dist</li>
    <li>python313Packages.tensordict</li>
    <li>python313Packages.tensordict.dist</li>
    <li>python313Packages.tensorflow-probability</li>
    <li>python313Packages.tensorflow-probability.dist</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.tianshou.dist</li>
    <li>python313Packages.treescope</li>
    <li>python313Packages.treescope.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vqgan-jax</li>
    <li>python313Packages.vqgan-jax.dist</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.waymax.dist</li>
    <li>python314Packages.anndata</li>
    <li>python314Packages.anndata.dist</li>
    <li>python314Packages.array-api-compat</li>
    <li>python314Packages.array-api-compat.dist</li>
    <li>python314Packages.augmax</li>
    <li>python314Packages.augmax.dist</li>
    <li>python314Packages.blackjax</li>
    <li>python314Packages.blackjax.dist</li>
    <li>python314Packages.chex</li>
    <li>python314Packages.chex.dist</li>
    <li>python314Packages.dm-haiku</li>
    <li>python314Packages.dm-haiku.dist</li>
    <li>python314Packages.equinox</li>
    <li>python314Packages.equinox.dist</li>
    <li>python314Packages.flowmc</li>
    <li>python314Packages.flowmc.dist</li>
    <li>python314Packages.funsor</li>
    <li>python314Packages.funsor.dist</li>
    <li>python314Packages.jax</li>
    <li>python314Packages.jax-cuda12-pjrt</li>
    <li>python314Packages.jax-cuda12-plugin</li>
    <li>python314Packages.jax.dist</li>
    <li>python314Packages.jaxlib (python314Packages.jaxlib-bin)</li>
    <li>python314Packages.jaxlib.dist (python314Packages.jaxlib-bin.dist)</li>
    <li>python314Packages.jaxopt</li>
    <li>python314Packages.jaxopt.dist</li>
    <li>python314Packages.jmp</li>
    <li>python314Packages.jmp.dist</li>
    <li>python314Packages.mujoco-mjx</li>
    <li>python314Packages.mujoco-mjx.dist</li>
    <li>python314Packages.nifty8</li>
    <li>python314Packages.nifty8.dist</li>
    <li>python314Packages.optax</li>
    <li>python314Packages.optax.dist</li>
    <li>python314Packages.optax.testsout</li>
    <li>python314Packages.orbax-checkpoint</li>
    <li>python314Packages.orbax-checkpoint.dist</li>
    <li>python314Packages.paramax</li>
    <li>python314Packages.paramax.dist</li>
    <li>python314Packages.pyvers</li>
    <li>python314Packages.pyvers.dist</li>
    <li>python314Packages.scikit-bio</li>
    <li>python314Packages.scikit-bio.dist</li>
    <li>python314Packages.tensordict</li>
    <li>python314Packages.tensordict.dist</li>
    <li>python314Packages.treescope</li>
    <li>python314Packages.treescope.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python313Packages.funsor</li>
    <li>python313Packages.funsor.dist</li>
    <li>python314Packages.funsor</li>
    <li>python314Packages.funsor.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 104 packages built:</summary>
  <ul>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>python313Packages.anndata</li>
    <li>python313Packages.anndata.dist</li>
    <li>python313Packages.array-api-compat</li>
    <li>python313Packages.array-api-compat.dist</li>
    <li>python313Packages.augmax</li>
    <li>python313Packages.augmax.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.chex</li>
    <li>python313Packages.chex.dist</li>
    <li>python313Packages.dm-haiku</li>
    <li>python313Packages.dm-haiku.dist</li>
    <li>python313Packages.equinox</li>
    <li>python313Packages.equinox.dist</li>
    <li>python313Packages.flowmc</li>
    <li>python313Packages.flowmc.dist</li>
    <li>python313Packages.funsor</li>
    <li>python313Packages.funsor.dist</li>
    <li>python313Packages.jax</li>
    <li>python313Packages.jax.dist</li>
    <li>python313Packages.jaxlib (python313Packages.jaxlib-bin)</li>
    <li>python313Packages.jaxlib.dist (python313Packages.jaxlib-bin.dist)</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.jmp</li>
    <li>python313Packages.jmp.dist</li>
    <li>python313Packages.lineax</li>
    <li>python313Packages.lineax.dist</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.optax</li>
    <li>python313Packages.optax.dist</li>
    <li>python313Packages.optax.testsout</li>
    <li>python313Packages.optimistix</li>
    <li>python313Packages.optimistix.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>python313Packages.orbax-export</li>
    <li>python313Packages.orbax-export.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.paramax</li>
    <li>python313Packages.paramax.dist</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pymanopt.dist</li>
    <li>python313Packages.pyvers</li>
    <li>python313Packages.pyvers.dist</li>
    <li>python313Packages.scikit-bio</li>
    <li>python313Packages.scikit-bio.dist</li>
    <li>python313Packages.tensordict</li>
    <li>python313Packages.tensordict.dist</li>
    <li>python313Packages.treescope</li>
    <li>python313Packages.treescope.dist</li>
    <li>python314Packages.anndata</li>
    <li>python314Packages.anndata.dist</li>
    <li>python314Packages.array-api-compat</li>
    <li>python314Packages.array-api-compat.dist</li>
    <li>python314Packages.augmax</li>
    <li>python314Packages.augmax.dist</li>
    <li>python314Packages.blackjax</li>
    <li>python314Packages.blackjax.dist</li>
    <li>python314Packages.chex</li>
    <li>python314Packages.chex.dist</li>
    <li>python314Packages.dm-haiku</li>
    <li>python314Packages.dm-haiku.dist</li>
    <li>python314Packages.equinox</li>
    <li>python314Packages.equinox.dist</li>
    <li>python314Packages.flowmc</li>
    <li>python314Packages.flowmc.dist</li>
    <li>python314Packages.funsor</li>
    <li>python314Packages.funsor.dist</li>
    <li>python314Packages.jax</li>
    <li>python314Packages.jax.dist</li>
    <li>python314Packages.jaxlib (python314Packages.jaxlib-bin)</li>
    <li>python314Packages.jaxlib.dist (python314Packages.jaxlib-bin.dist)</li>
    <li>python314Packages.jaxopt</li>
    <li>python314Packages.jaxopt.dist</li>
    <li>python314Packages.jmp</li>
    <li>python314Packages.jmp.dist</li>
    <li>python314Packages.lineax</li>
    <li>python314Packages.lineax.dist</li>
    <li>python314Packages.minari</li>
    <li>python314Packages.minari.dist</li>
    <li>python314Packages.nifty8</li>
    <li>python314Packages.nifty8.dist</li>
    <li>python314Packages.optax</li>
    <li>python314Packages.optax.dist</li>
    <li>python314Packages.optax.testsout</li>
    <li>python314Packages.optimistix</li>
    <li>python314Packages.optimistix.dist</li>
    <li>python314Packages.orbax-checkpoint</li>
    <li>python314Packages.orbax-checkpoint.dist</li>
    <li>python314Packages.paramax</li>
    <li>python314Packages.paramax.dist</li>
    <li>python314Packages.pyvers</li>
    <li>python314Packages.pyvers.dist</li>
    <li>python314Packages.scikit-bio</li>
    <li>python314Packages.scikit-bio.dist</li>
    <li>python314Packages.tensordict</li>
    <li>python314Packages.tensordict.dist</li>
    <li>python314Packages.treescope</li>
    <li>python314Packages.treescope.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>vllm</summary>
<pre>    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File &quot;/nix/store/jldwaqq2p69ms46wpf74aw5rllnhwz6j-python3.13-setuptools-80.10.1/lib/python3.13/site-packages/setuptools/dist.py&quot;, line 1107, in run_command
    super().run_command(command)
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File &quot;/nix/store/jldwaqq2p69ms46wpf74aw5rllnhwz6j-python3.13-setuptools-80.10.1/lib/python3.13/site-packages/setuptools/_distutils/dist.py&quot;, line 1021, in run_command
    cmd_obj.run()
    ~~~~~~~~~~~^^
  File &quot;&lt;string&gt;&quot;, line 370, in run
  File &quot;/nix/store/jldwaqq2p69ms46wpf74aw5rllnhwz6j-python3.13-setuptools-80.10.1/lib/python3.13/site-packages/setuptools/command/build_ext.py&quot;, line 97, in run
    _build_ext.run(self)
    ~~~~~~~~~~~~~~^^^^^^
  File &quot;/nix/store/jldwaqq2p69ms46wpf74aw5rllnhwz6j-python3.13-setuptools-80.10.1/lib/python3.13/site-packages/setuptools/_distutils/command/build_ext.py&quot;, line 368, in run
    self.build_extensions()
    ~~~~~~~~~~~~~~~~~~~~~^^
  File &quot;&lt;string&gt;&quot;, line 339, in build_extensions
  File &quot;/nix/store/pzdalg368npikvpq4ncz2saxnz19v53k-python3-3.13.12/lib/python3.13/subprocess.py&quot;, line 419, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command &#x27;[&#x27;cmake&#x27;, &#x27;--build&#x27;, &#x27;.&#x27;, &#x27;-j=64&#x27;, &#x27;--target=_C&#x27;]&#x27; returned non-zero exit status 1.

ERROR Backend subprocess exited when trying to invoke build_wheel</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>vllm</summary>
<pre>/bin/python3.13&#x27;, &#x27;-DVLLM_PYTHON_PATH=/nix/store/nnz5k19wbai97d7y2x11qaq6dgxzhq4w-python3-3.13.12/lib/python3.13/site-packages:/nix/store/83fg8010nrh29dihrd6rbyaaambfsnf0-python3.13-wheel-0.46.1/lib/python3.13/site-packages:/nix/store/3b1wb33pi5bxdks5acr2a3h7bzic3py0-python3.13-packaging-25.0/lib/python3.13/site-packages:/nix/store/v61d2dsam178hxm9n988vgs1z1vm6nqg-python3.13-installer-0.7.0/lib/python3.13/site-packages:/nix/store/diqn19xcmv2asczp6cd7c7dkv814ns9p-python3.13-cmake-4.1.2/lib/python3.13/site-packages:/nix/store/lsv6203d7ipaagy67g3nbvvn9apikwh5-python3.13-protobuf-7.34.0/lib/python3.13/site-packages:/nix/store/m5pcggas3k136avj8xb9yhzkg1zrwsbn-python3.13-typing-extensions-4.15.0/lib/python3.13/site-packages:/nix/store/yvyv8vv0f52xg1zyqi1m8mpxglgp4dkj-python3.13-grpcio-1.78.0/lib/python3.13/site-packages:/nix/store/fjd5js1p4z44flyi9c59a6vrl0qsk1cp-python3.13-setuptools-80.10.1/lib/python3.13/site-packages:/nix/store/7116rv6h6hmz08429z9l7pb7x6fvw8sb-python3.13-grpcio-tools-1.78.0/lib/python3.13/site-packages:/nix/store/y7vq12q4fljjamc39pfv83j5xj29rprz-python3.13-jinja2-3.1.6/lib/python3.13/site-packages:/nix/store/mznm0jdcad3cbkkdbd31fvk147sviiqr-python3.13-markupsafe-3.0.3/lib/python3.13/site-packages:/nix/store/w6vyqvzvkagxd6zyfifgic3va80aqxq0-python3.13-ninja-1.13.2/lib/python3.13/site-packages:/nix/store/a0ffymds5w97pcir0rzvlbnaavpxjfkv-python3.13-setuptools-scm-9.2.2/lib/python3.13/site-packages:/nix/store/bvkxxljx8vnfjz54605dyix8fg5kpn4y-python3.13-astunparse-1.6.3/lib/python3.13/site-packages:/nix/store/3xm286v7d8433bf3dmcg00lf6fn4drqy-python3.13-six-1.17.0/lib/python3.13/site-packages:/nix/store/alzdx31x91nc3y5bnb5zxy7rv7f9r8sd-python3.13-expecttest-0.3.0/lib/python3.13/site-packages:/nix/store/2n6q9dswbqknamqiqf8fmdihr6ny9050-python3.13-filelock-3.20.3/lib/python3.13/site-packages:/nix/store/50iify42ap7iwb1681algcfgw1y0xigi-python3.13-fsspec-2026.1.0/lib/python3.13/site-packages:/nix/store/bmvljghibqnaw3acz9as712szzpxw6il-python3.13-hypothesis-6.150.2/lib/python3.13/site-packages:/nix/store/hh3164gn45arniiq0wdv1qxf7f0bxalh-python3.13-attrs-25.4.0/lib/python3.13/site-packages:/nix/store/7h6fzllk0p6kjic1hrdj3jh5p804p6hd-python3.13-sortedcontainers-2.4.0/lib/python3.13/site-packages:/nix/store/fny3yn8dwd3m9j89f11r6a35nbn3m2j3-python3.13-networkx-3.6.1/lib/python3.13/site-packages:/nix/store/n1hpzy8fp76y8n643hc3s98xnfmhmlr8-python3.13-psutil-7.2.1/lib/python3.13/site-packages:/nix/store/2vk12grbja9kgal0d4wx0z163pn5g6rj-python3.13-pyyaml-6.0.3/lib/python3.13/site-packages:/nix/store/s8mjykmxhnahk83nr9mww6ybp9g7wwj5-python3.13-requests-2.32.5/lib/python3.13/site-packages:/nix/store/xksisvgjx0y6v75mq10ijy9xp3yl3kh7-python3.13-certifi-2026.01.04/lib/python3.13/site-packages:/nix/store/283b7p3dw2cq45wsvsdfmgkxhz22hcjn-python3.13-charset-normalizer-3.4.4/lib/python3.13/site-packages:/nix/store/62rb271fbnzc8a0j8sk5kp08qy8gl66k-python3.13-idna-3.11/lib/python3.13/site-packages:/nix/store/8xz711h7gfc7152an6s5q138wg4vzvlp-python3.13-urllib3-2.6.3/lib/python3.13/site-packages:/nix/store/88zzfnlrabjbi8spphxjcik9dbd8dlz8-python3.13-sympy-1.14.0/lib/python3.13/site-packages:/nix/store/1v4b940434k875asm5a1h9wcpb6ls01z-python3.13-mpmath-1.3.0/lib/python3.13/site-packages:/nix/store/lp40x0d4vqh1ym1w33h66382pg0vsnj6-python3.13-types-dataclasses-0.6.6/lib/python3.13/site-packages:/nix/store/4jyxjsw0pg43pn6rh1f1h0rgglvi4spl-python3.13-pillow-12.1.1/lib/python3.13/site-packages:/nix/store/zfygf97hqrswq5kkggffjjs91j0gilj5-python3.13-tensorboard-2.20.0/lib/python3.13/site-packages:/nix/store/17cdmjxbd1mdmifgqji1zhfxdzf2dcbg-python3.13-absl-py-2.3.1/lib/python3.13/site-packages:/nix/store/2v9lmx7i3vr23y9y6922wvab6ykmznvx-python3.13-markdown-3.10.2/lib/python3.13/site-packages:/nix/store/crfqkslv3jlla8wff130la2fjim01ld9-python3.13-numpy-2.4.2/lib/python3.13/site-packages:/nix/store/yjcxgc0m5hp470kliqf0yabdxba0h9m3-python3.13-tensorboard-data-server-0.7.2/lib/python3.13/site-packages:/nix/store/4yimmis812r5myd8412lyf8rcs4y3g5q-python3.13-werkzeug-3.1.6/lib/python3.13/site-packages:/nix/store/36ci0rm24wpbxg0kvv90nf6fx2c9p6qw-python3.13-standard-imghdr-3.13.0/lib/python3.13/site-packages:/nix/store/8db6ajvm4bjjlld0rpdwxflzc4y6a57g-python3.13-pybind11-3.0.1/lib/python3.13/site-packages:/nix/store/vpmgvy3qrj04y7hr7sj2nwkmqrj2bwf4-python3.13-triton-3.6.0/lib/python3.13/site-packages:/nix/store/ls1q77702lr18891k4f4b77bbv4riyvk-python3.13-torch-2.11.0/lib/python3.13/site-packages:/nix/store/6n9glqadp4ad1v98x359b7kfc502489g-python3.13-cffi-2.0.0/lib/python3.13/site-packages:/nix/store/bmflhr5pj0gg1r2vrbm7qzpjimzj8xp6-python3.13-pycparser-3.00/lib/python3.13/site-packages:/nix/store/cp3a84ck7hfs5qg6rhq3i0p50j2xzmjx-python3.13-aioprometheus-unstable-2023-03-14/lib/python3.13/site-packages:/nix/store/zzwzsrwawn16b7a3if83ms9i8hd391ql-python3.13-orjson-3.11.5/lib/python3.13/site-packages:/nix/store/2ggj83jfr84d4p2anqz92zd6nvfwc427-python3.13-quantile-python-1.1/lib/python3.13/site-packages:/nix/store/v7w8r9l29ab4wkf8lwpkvvyrs3v1cz3y-python3.13-anthropic-0.94.0/lib/python3.13/site-packages:/nix/store/4kfdxd7w2lbnwwdms3m49kdrip9yi1jm-python3.13-anyio-4.12.1/lib/python3.13/site-packages:/nix/store/vr9kaqqf1a6r9c9b3qnj514wblsldc3m-python3.13-distro-1.9.0/lib/python3.13/site-packages:/nix/store/6mzrvg70rxq885c62z4jf9fxs3zh1l9v-python3.13-docstring-parser-0.17.0/lib/python3.13/site-packages:/nix/store/y1njynxbbmsr7cpbn9hfvkf00sxkbvps-python3.13-httpx-0.28.1/lib/python3.13/site-packages:/nix/store/7a4jgl6m16j1i3c8iv2j1lksblr2r2dl-python3.13-httpcore-1.0.9/lib/python3.13/site-packages:/nix/store/mhahf0wa8bwqjlnbpnc9wv0ban1x2m2d-python3.13-h11-0.16.0/lib/python3.13/site-packages:/nix/store/m98wgy9w43mkb9pkcm808bdm2nv833ig-python3.13-jiter-0.12.0/lib/python3.13/site-packages:/nix/store/2sh17bkdn3vd6hjld0agfyr9pvppbk8k-python3.13-pydantic-2.12.5/lib/python3.13/site-packages:/nix/store/h73gzkwblph3s3sj1j7lg7fy99xicf1i-python3.13-annotated-types-0.7.0/lib/python3.13/site-packages:/nix/store/5zy9kdvv8ck06xmw3fvnhm82wkcan4mi-python3.13-pydantic-core-2.41.5/lib/python3.13/site-packages:/nix/store/11mcvn18slr5d375qziknkw0mvlkqvl9-python3.13-typing-inspection-0.4.2/lib/python3.13/site-packages:/nix/store/9cw086hmav66jwnx1s3hl74bvd7mgr6h-python3.13-sniffio-1.3.1/lib/python3.13/site-packages:/nix/store/qcwvd09m7n3dlv0vsnlw5i9i4rsiya77-python3.13-bitsandbytes-0.49.2/lib/python3.13/site-packages:/nix/store/a12khdy18hbrwk5kjrrl58sm440b4hkm-python3.13-scipy-1.17.1/lib/python3.13/site-packages:/nix/store/73qvirs9q1ymak5chj26z2wnbpahlp88-python3.13-trove-classifiers-2026.1.14.14/lib/python3.13/site-packages:/nix/store/qwngdg3flkm17bjd1ydzby76xkycndnf-python3.13-blake3-1.0.8/lib/python3.13/site-packages:/nix/store/f5bj2zzlbdv5ja4ahmadrpysbww5kwbv-python3.13-cachetools-6.2.4/lib/python3.13/site-packages:/nix/store/14q9sdnicxfbgkjgfhbb7ki40h5zmgn4-python3.13-cbor2-5.8.0/lib/python3.13/site-packages:/nix/store/4jjdzfabh9qj9dxaham6va2j80zsvhg2-python3.13-compressed-tensors-0.15.0.1/lib/python3.13/site-packages:/nix/store/9b5983h43dkj6l1724gvimkzshdn0n8z-python3.13-frozendict-2.4.7/lib/python3.13/site-packages:/nix/store/inji7mlhlvh6rxmk7z78hx0h2j9d62ac-python3.13-loguru-0.7.3/lib/python3.13/site-packages:/nix/store/gfpl9fmya8m8k0dmyplrlkjsya6vkbq5-python3.13-transformers-5.5.4/lib/python3.13/site-packages:/nix/store/sbhl7f8hxsk04112vjdcda3075yj197v-python3.13-huggingface-hub-1.10.2/lib/python3.13/site-packages:/nix/store/ig1p9vzp0960d6za08r918gjp1hdif23-python3.13-hf-xet-1.4.3/lib/python3.13/site-packages:/nix/store/80f4hnfqmfkn6rillfgpqf2c3jgl5i10-python3.13-tqdm-4.67.1/lib/python3.13/site-packages:/nix/store/0abzf4k988ryzy5mglinp99bz8f9gfnc-python3.13-typer-0.24.0/lib/python3.13/site-packages:/nix/store/d3x3labrckm85mqvsd8nzsqv20y0dji6-python3.13-annotated-doc-0.0.4/lib/python3.13/site-packages:/nix/store/58szw0p3i10nicrmip2h1zv2syv14csz-python3.13-click-8.3.1/lib/python3.13/site-packages:/nix/store/x04c8m9k5bkl805ffq16nck5dp5glnw4-python3.13-rich-14.3.3/lib/python3.13/site-packages:/nix/store/h157s4i3fcn3mgghp8nvpj4zj7rc2czb-python3.13-markdown-it-py-4.0.0/lib/python3.13/site-packages:/nix/store/z4kq0yp27wsn33jjlkzr84zddrs1izyh-python3.13-mdurl-0.1.2/lib/python3.13/site-packages:/nix/store/g1qbidyq42k985jckf6lvr2dbpl8ac8p-python3.13-pygments-2.19.2/lib/python3.13/site-packages:/nix/store/i06gr3dv2dml63i793iplvwvkp2l8bjk-python3.13-shellingham-1.5.4/lib/python3.13/site-packages:/nix/store/2ppjj4w42z7h94s0y6s0wigbkxxf01f1-python3.13-regex-2026.2.19/lib/python3.13/site-packages:/nix/store/9yynpx0vr8b67fdpn47rnsp2lkyy6fjy-python3.13-safetensors-0.7.0/lib/python3.13/site-packages:/nix/store/1vx9mp1dwr3zc3ghs35sd5wdlh8frl8h-python3.13-tokenizers-0.22.2/lib/python3.13/site-packages:/nix/store/mb15qjji6qjw3axdlfpsbpr28asalwwi-python3.13-depyf-0.20.0/lib/python3.13/site-packages:/nix/store/wr4vk2qqbwpmsx2vjaq066nwshczhvj6-python3.13-astor-0.8.1-unstable-2024-03-30/lib/python3.13/site-packages:/nix/store/fj9xa291swz4sqn80x7xzjfabbazj16v-python3.13-dill-0.4.1/lib/python3.13/site-packages:/nix/store/l4qgkq6snmnbkqbylnlv8whbf8lf34hj-python3.13-einops-0.8.1/lib/python3.13/site-packages:/nix/store/10igliqgvy752yqhswqcbgy16nmpvfy2-python3.13-fastapi-0.128.0/lib/python3.13/site-packages:/nix/store/c9vw014h5ijdj99zygz10mcswms1s32r-python3.13-starlette-0.52.1/lib/python3.13/site-packages:/nix/store/mivg7kfnp5m705wc119hyr7ycirddqas-python3.13-gguf-8799/lib/python3.13/site-packages:/nix/store/m71n72vdgairddmkwsva62f14ld85gkh-python3.13-grpcio-reflection-1.78.0/lib/python3.13/site-packages:/nix/store/p404isk3hi67xvyrfrqb7l3wzxkzkd0w-python3.13-ijson-3.5.0/lib/python3.13/site-packages:/nix/store/vaz9r31dyvxvqdbi0xfcabgimi2v2cgi-python3.13-importlib-metadata-8.7.1/lib/python3.13/site-packages:/nix/store/wsbv8p4bgm41hsbiqwaarv13zwbw2jqb-python3.13-toml-0.10.2/lib/python3.13/site-packages:/nix/store/20azcig0fkv0sbawysmlsv9zapiby62i-python3.13-zipp-3.23.1/lib/python3.13/site-packages:/nix/store/xd00qxpcw2bvvlyk95k6m7b7sn9xi0r7-python3.13-llguidance-1.7.2/lib/python3.13/site-packages:/nix/store/pa29b5vbl677n914vfiv1crp1rvwgipj-python3.13-lm-format-enforcer-0.11.3/lib/python3.13/site-packages:/nix/store/1143q6j63d6xfgr40dbp204vig24bxmz-python3.13-interegular-0.3.3/lib/python3.13/site-packages:/nix/store/d8f4zhv6g20kh3ri2zj9mhlnmwldw8a3-python3.13-mcp-1.26.0/lib/python3.13/site-packages:/nix/store/dxc3iq763y2nriwjnnd448hj8by0p9q2-python3.13-httpx-sse-0.4.3/lib/python3.13/site-packages:/nix/store/n03d248x1n1259aky3b3gaiby6cyv46y-python3.13-jsonschema-4.26.0/lib/python3.13/site-packages:/nix/store/7322gfi6v3z5z7n2w2yasrmnzbdh8w31-python3.13-jsonpath-ng-1.7.0/lib/python3.13/site-packages:/nix/store/znbgx090d3gkpdhsmdkcna12mznszg7l-python3.13-ply-3.11/lib/python3.13/site-packages:/nix/store/ynm7fqahkmwlxz5pwifjllpbp8nmy317-python3.13-jsonschema-specifications-2025.9.1/lib/python3.13/site-packages:/nix/store/93zw61r406fqzcgyhkngfkhvvkl3qa1v-python3.13-referencing-0.37.0/lib/python3.13/site-packages:/nix/store/zbzd1y6bhsvx0n1a44rfnlpfqd1j8nx2-python3.13-rpds-py-0.30.0/lib/python3.13/site-packages:/nix/store/cyfnsypa9fjjhy65vai428g4krd8laj4-python3.13-pydantic-settings-2.12.0/lib/python3.13/site-packages:/nix/store/kxm4p6ljksa9dzsh8vgr56j6r8pvwnin-python3.13-python-dotenv-1.2.1/lib/python3.13/site-packages:/nix/store/w00rn7f11w0b63b6r749rsdg9zzxqr48-python3.13-pyjwt-2.11.0/lib/python3.13/site-packages:/nix/store/z1w6d6iyws2y3pv62p2g7cnv01jpkhbq-python3.13-python-multipart-0.0.21/lib/python3.13/site-packages:/nix/store/0y08v5vcgp1ning7br1c62c0lvyjlk9v-python3.13-sse-starlette-3.2.0/lib/python3.13/site-packages:/nix/store/r9x38zcfrl3jxyh43y7531q4b8qbyhv0-python3.13-uvicorn-0.40.0/lib/python3.13/site-packages:/nix/store/51hbx093vzz3b5kqkz1gsvq08b37da16-python3.13-mistral-common-1.8.8/lib/python3.13/site-packages:/nix/store/szzzlgcxa5dgjy2ys657ghjl9shdmpkq-python3.13-opencv-python-headless-4.13.0/lib/python3.13/site-packages:/nix/store/5achm800hlv289ybng8ab97r8xdqj0kf-opencv-4.13.0/lib/python3.13/site-packages:/nix/store/nflkxq01215cq2hiy4hyjsp3g253ix3r-python3.13-pydantic-extra-types-2.11.0/lib/python3.13/site-packages:/nix/store/r67kc6vdha0h4sfjalikcqsc3vam5kvx-python3.13-sentencepiece-0.2.1/lib/python3.13/site-packages:/nix/store/f1ai6lk648vswymr653a1q6rk0ivf5zw-python3.13-tiktoken-0.12.0/lib/python3.13/site-packages:/nix/store/z62kzyxa2afsf8r14y843nh4gl4ln311-python3.13-blobfile-3.1.0/lib/python3.13/site-packages:/nix/store/5l4g68xdmsqy5vdjnlafdh8hsp17ff01-python3.13-pycryptodomex-3.23.0/lib/python3.13/site-packages:/nix/store/j8qyvpq656vj85pwdcvn18zvl7a7yvc0-python3.13-lxml-6.0.2/lib/python3.13/site-packages:/nix/store/gw09by4qqk6llqnv832wn4ivsayk6hd2-python3.13-model-hosting-container-standards-0.1.14/lib/python3.13/site-packages:/nix/store/10izdcr8vhwxnhcq7gqz084znn70n29x-python3.13-jmespath-1.0.1/lib/python3.13/site-packages:/nix/store/6w00gwkjjj47j3ddilr35hma8z3jmnh2-python3.13-supervisor-4.3.0/lib/python3.13/site-packages:/nix/store/hyzxq2v98yzx5sx291jcpcpcfw2jg8l4-python3.13-msgspec-0.20.0/lib/python3.13/site-packages:/nix/store/x0d6fi74l6j65gwklpcswzyfyznhi6il-python3.13-numba-0.63.1/lib/python3.13/site-packages:/nix/store/v77m6iypkjsr8msnn0wg4hjxr2av5xcz-python3.13-llvmlite-0.46.0/lib/python3.13/site-packages:/nix/store/jvbqscs2gb8vvd1w5nrcvr6255ymbr3d-python3.13-openai-2.29.0/lib/python3.13/site-packages:/nix/store/mxiic1yks9ld0k1rqg0cix27s31bymr8-python3.13-aiohttp-3.13.3/lib/python3.13/site-packages:/nix/store/3bplp57h85rnmmvh6ysahsld3dnh9rvz-python3.13-aiohappyeyeballs-2.6.1/lib/python3.13/site-packages:/nix/store/nggkkx0v9kj0ami4xkfz345cdrbqaaz9-python3.13-aiosignal-1.4.0/lib/python3.13/site-packages:/nix/store/jdm1m36id090f9r5h3ag9vh16v6rc6yd-python3.13-frozenlist-1.8.0/lib/python3.13/site-packages:/nix/store/9ywsyplf9igbpx0mvxa7s4z7x98bbfap-python3.13-multidict-6.7.1/lib/python3.13/site-packages:/nix/store/7imqcmig7sv3lp10q1i9r85yxx25dmsj-python3.13-propcache-0.4.1/lib/python3.13/site-packages:/nix/store/g5cc6db5fj61b0i9y7pw3k29fd7sn6ys-python3.13-yarl-1.23.0/lib/python3.13/site-packages:/nix/store/zn8317mkyigi9skzf42fy78p69lfgdq9-python3.13-aiodns-4.0.0/lib/python3.13/site-packages:/nix/store/x344rcvqxyl54wk5mng59hfrx1im4vig-python3.13-pycares-5.0.1/lib/python3.13/site-packages:/nix/store/rxa50vqdi74i0l6rbfy0smrqsrkvmmh1-python3.13-brotli-1.2.0/lib/python3.13/site-packages:/nix/store/xw6b8z7ymnkbc3fc688h9s3i3di0ldjf-python3.13-backports-zstd-1.3.0/lib/python3.13/site-packages:/nix/store/i016jbnlk209lsln5id7y5hyfdzanar1-python3.13-httpx-aiohttp-0.1.12/lib/python3.13/site-packages:/nix/store/iz49qyqsqyshwp8bsnkzqp4i5ppwyw3n-python3.13-websockets-16.0/lib/python3.13/site-packages:/nix/store/p0k4cvc8ah3daqa0gm6krv1ywc8iv1ly-python3.13-sounddevice-0.5.3/lib/python3.13/site-packages:/nix/store/smky4vf9mqq347sim9hndjilm3i9plp6-python3.13-openai-harmony-0.0.8/lib/python3.13/site-packages:/nix/store/7hhzachwp2125nbwwl80lb0r8rlvwasf-python3.13-opentelemetry-api-1.34.0/lib/python3.13/site-packages:/nix/store/01qknkwzxzi1rh0dwrfiganjxzhapjy8-python3.13-deprecated-1.3.1/lib/python3.13/site-packages:/nix/store/hfygb43mnhfpgkfkpgxx18ik8f0piz1c-python3.13-wrapt-1.17.2/lib/python3.13/site-packages:/nix/store/2y4i57crggygqv73spbf030lr1b8y0sk-python3.13-opentelemetry-exporter-otlp-1.34.0/lib/python3.13/site-packages:/nix/store/3hv95sjmx4r1shclxqikzp0qpp63bf20-python3.13-opentelemetry-exporter-otlp-proto-grpc-1.34.0/lib/python3.13/site-packages:/nix/store/p4mzkkvycaszmcsgq0ccyzirgdc8d1r5-python3.13-googleapis-common-protos-1.73.0/lib/python3.13/site-packages:/nix/store/yqs9nxpppf2zf2nc3gw77aqfkch3zkjf-python3.13-opentelemetry-exporter-otlp-proto-common-1.34.0/lib/python3.13/site-packages:/nix/store/hxily3xhbwax5nc82nkz4vgpvxwykgdg-python3.13-opentelemetry-proto-1.34.0/lib/python3.13/site-packages:/nix/store/98xwjf9vhv5qah4dm7xlw7p62bc2qj3p-python3.13-opentelemetry-exporter-otlp-proto-http-1.34.0/lib/python3.13/site-packages:/nix/store/xzr668g8vpad1iigyjrfd2njg4477l0p-python3.13-opentelemetry-sdk-1.34.0/lib/python3.13/site-packages:/nix/store/p29ldsvwrhq7bwhb6j5316095vjmi4p6-python3.13-opentelemetry-semantic-conventions-0.55b0/lib/python3.13/site-packages:/nix/store/l73yqvgkib8acnwlygxb6kkcpfih9riq-python3.13-outlines-1.2.12/lib/python3.13/site-packages:/nix/store/7cwjl6awpn16kfz9zzfcdnvmw5qy4xnc-python3.13-cloudpickle-3.1.2/lib/python3.13/site-packages:/nix/store/9gky6hggakxx9p4awm4jgx7df6pp8gl8-python3.13-datasets-4.5.0/lib/python3.13/site-packages:/nix/store/hl2pfiyg9wgyqym810jcqby0mqf8rjk3-python3.13-multiprocess-0.70.19/lib/python3.13/site-packages:/nix/store/7n2126n92956309x5ahyn1r6lgh7qkqc-python3.13-pandas-2.3.3/lib/python3.13/site-packages:/nix/store/q6w1r871xs80v8nk82ld0csm7ixzhr7l-python3.13-python-dateutil-2.9.0.post0/lib/python3.13/site-packages:/nix/store/krm1m7vgw8ad25zq8m3mzsl0mfh49yn8-python3.13-pytz-2026.1.post1/lib/python3.13/site-packages:/nix/store/6fq0kwxc8cyzzpp1hgzwk8d6qi6j39n1-python3.13-tzdata-2025.3/lib/python3.13/site-packages:/nix/store/gyhjf171y1i9lyji0xnxp5agygh92nny-python3.13-pyarrow-23.0.0/lib/python3.13/site-packages:/nix/store/r1ahh4l3vjkpq443vndc5hqr170j1rrm-python3.13-xxhash-3.6.0/lib/python3.13/site-packages:/nix/store/y536r84mjh5bgrrqxah27v5ihp52qqa0-python3.13-diskcache-5.6.3/lib/python3.13/site-packages:/nix/store/2nzrlgk82yb9qrgvg39cscwc0w4pkdh1-python3.13-genson-1.3.0/lib/python3.13/site-packages:/nix/store/0p7xgp82ich9sy1pv2rmj2z5r5rb2bmp-python3.13-lark-1.3.1/lib/python3.13/site-packages:/nix/store/c8m53gx6zv0s7blmlg0035ijddy6265m-python3.13-nest-asyncio-1.6.0/lib/python3.13/site-packages:/nix/store/cbkd2q61yqhmd1fzb5hj15ikl31fi7p1-python3.13-outlines-core-0.2.14/lib/python3.13/site-packages:/nix/store/3fdb7nwl62cqf2hpp1k3glxixl4kjjkb-python3.13-pycountry-26.2.16/lib/python3.13/site-packages:/nix/store/5gkjinnfmdxaa9x7z05fji4px93b8gd0-python3.13-partial-json-parser-0.2.1.1.post7/lib/python3.13/site-packages:/nix/store/ja972wbfgw062pdkdd8aiqk175p6162m-python3.13-prometheus-fastapi-instrumentator-7.1.0/lib/python3.13/site-packages:/nix/store/67991l3k9sip7safvmfxjw83zxzqd63c-python3.13-prometheus-client-0.24.1/lib/python3.13/site-packages:/nix/store/5s6cp3nn7m5lsg9ww5kfyp702m1wkhbj-python3.13-asgiref-3.11.0/lib/python3.13/site-packages:/nix/store/f56144s4mz6c6bq3zvvd06n6l2mjs53n-python3.13-py-cpuinfo-9.0.0-unstable-2022-11-20/lib/python3.13/site-packages:/nix/store/83br49kd5866v86mbhajw5znahspwxam-python3.13-pybase64-1.4.3/lib/python3.13/site-packages:/nix/store/fhkssmj5jwsiw0nd2iqyidpgp00g4rzc-python3.13-python-json-logger-4.0.0/lib/python3.13/site-packages:/nix/store/8hqy47455zxv9n16s0f3qxc2490r5d8y-python3.13-pyzmq-27.1.0/lib/python3.13/site-packages:/nix/store/lrrxg9a9wr0h14mb42znr10ix4y79ldf-python3.13-ray-2.55.0/lib/python3.13/site-packages:/nix/store/v4jdwy8d60m854rc6qav2k4ifd8h4n5c-python3.13-msgpack-1.1.2/lib/python3.13/site-packages:/nix/store/7cpsp1kcvhma5k9dc548n0hrf5935fcv-python3.13-setproctitle-1.3.7/lib/python3.13/site-packages:/nix/store/69dwldljwd3idrcpf6p6i0xchimvxa0i-python3.13-torchaudio-2.11.0/lib/python3.13/site-packages:/nix/store/lhp4kxnpnnpjd2s94slbjgki22pxxqkg-python3.13-torchcodec-0.11.1/lib/python3.13/site-packages:/nix/store/0m20zc7ziqr02k2crcmm536mpx4b4hb4-python3.13-torchvision-0.26.0/lib/python3.13/site-packages:/nix/store/19mvvsz7w1igz7w7y8rbkv9p2h5bk3xs-python3.13-xformers-0.0.35/lib/python3.13/site-packages:/nix/store/h6hjrjclfkzglfc56p5jiidxy72hidrn-python3.13-xgrammar-0.1.33/lib/python3.13/site-packages:/nix/store/a1jq21gnfd4v1sf9ff5bmni09ldrbszm-python3.13-httptools-0.7.1/lib/python3.13/site-packages:/nix/store/h1q9l27y1plwsgbk4cfvnws1hrkcxr1n-python3.13-uvloop-0.22.1/lib/python3.13/site-packages:/nix/store/xyl69rl3ghprqp59dvmf0527prbcjibv-python3.13-watchfiles-1.1.1/lib/python3.13/site-packages:/nix/store/xjyyydp5ly97khf05ps2dh2my02dfi9h-python3.13-py-libnuma-1.2/lib/python3.13/site-packages:/nix/store/nnz5k19wbai97d7y2x11qaq6dgxzhq4w-python3-3.13.12/lib/python313.zip:/nix/store/nnz5k19wbai97d7y2x11qaq6dgxzhq4w-python3-3.13.12/lib/python3.13:/nix/store/nnz5k19wbai97d7y2x11qaq6dgxzhq4w-python3-3.13.12/lib/python3.13/lib-dynload:/nix/store/fjd5js1p4z44flyi9c59a6vrl0qsk1cp-python3.13-setuptools-80.10.1/lib/python3.13/site-packages/setuptools/_vendor:/nix/store/7116rv6h6hmz08429z9l7pb7x6fvw8sb-python3.13-grpcio-tools-1.78.0/lib/python3.13/site-packages/grpc_tools/_proto&#x27;, &#x27;-DFETCHCONTENT_BASE_DIR=/build/source/.deps&#x27;, &#x27;-DCMAKE_JOB_POOL_COMPILE:STRING=compile&#x27;, &#x27;-DCMAKE_JOB_POOLS:STRING=compile=20&#x27;]&#x27; returned non-zero exit status 1.

ERROR Backend subprocess exited when trying to invoke build_wheel</pre>
</details>
<details>
<summary>python314Packages.lineax</summary>
<pre>E                       equinox.EquinoxRuntimeError: Condition number of A seems to be larger than `conlim`.
E                       
E                       -------------------
E                       
E                       An error occurred during the runtime of your JAX program.
E                       
E                       1) Setting the environment variable `EQX_ON_ERROR=breakpoint` is usually the most useful
E                       way to debug such errors. This can be interacted with using most of the usual commands
E                       for the Python debugger: `u` and `d` to move up and down frames, the name of a variable
E                       to print its value, etc.
E                       
E                       2) You may also like to try setting `JAX_DISABLE_JIT=1`. This will mean that you can
E                       (mostly) inspect the state of your program as if it was normal Python.
E                       
E                       3) See `https://docs.kidger.site/equinox/api/debug/` for more suggestions.

tests/test_vmap_jvp.py:129: EquinoxRuntimeError
=========================== short test summary info ============================
FAILED tests/test_vmap_jvp.py::test_vmap_jvp[float64-construct_singular_matrix-False-make_jac_operator-solver17-tags17-True] - EquinoxRuntimeError: Above is the stack outside of JIT. Below is the stack ...
================== 1 failed, 335 passed in 1218.23s (0:20:18) ==================</pre>
</details>
</details>

--- a/pkgs/development/python-modules/chex/default.nix
+++ b/pkgs/development/python-modules/chex/default.nix
@@ -20,15 +20,16 @@
   pytestCheckHook,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage (finalAttrs: {
   pname = "chex";
   version = "0.1.91";
   pyproject = true;
+  __structuredAttrs = true;
 
   src = fetchFromGitHub {
     owner = "deepmind";
     repo = "chex";
-    tag = "v${version}";
+    tag = "v${finalAttrs.version}";
     hash = "sha256-lJ9+kvG7dRtfDVgvkcJ9/jtnX0lMfxY4mmZ290y/74U=";
   };
 
@@ -36,9 +37,6 @@ buildPythonPackage rec {
     flit-core
   ];
 
-  pythonRelaxDeps = [
-    "typing_extensions"
-  ];
   dependencies = [
     absl-py
     jax
@@ -69,8 +67,8 @@ buildPythonPackage rec {
   meta = {
     description = "Library of utilities for helping to write reliable JAX code";
     homepage = "https://github.com/deepmind/chex";
-    changelog = "https://github.com/google-deepmind/chex/releases/tag/v${version}";
+    changelog = "https://github.com/google-deepmind/chex/releases/tag/${finalAttrs.src.tag}";
     license = lib.licenses.asl20;
     maintainers = with lib.maintainers; [ ndl ];
   };
-}
+})

--- a/pkgs/development/python-modules/equinox/default.nix
+++ b/pkgs/development/python-modules/equinox/default.nix
@@ -21,14 +21,15 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "equinox";
-  version = "0.13.6";
+  version = "0.13.7";
   pyproject = true;
+  __structuredAttrs = true;
 
   src = fetchFromGitHub {
     owner = "patrick-kidger";
     repo = "equinox";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-OETWXAcCp945mMrpC8U4gSBvEeQX8RoUGZR4irBs7Ak=";
+    hash = "sha256-vgmU8cqNCyiZYah1SSwzVtLS+YB2T1uooCC17k12+h8=";
   };
 
   # Relax speed constraints on tests that can fail on busy builders
@@ -54,25 +55,6 @@ buildPythonPackage (finalAttrs: {
     optax
     pytest-xdist
     pytestCheckHook
-  ];
-
-  pytestFlags = [
-    # DeprecationWarning: The default axis_types will change in JAX v0.9.0 to jax.sharding.AxisType.Explicit.
-    "-Wignore::DeprecationWarning"
-  ];
-
-  disabledTestPaths = [
-    # ValueError: not enough values to unpack (expected 2, got 0)
-    "tests/test_finalise_jaxpr.py"
-  ];
-
-  disabledTests = [
-    # Failed: DID NOT WARN. No warnings of type (<class 'Warning'>,) were emitted.
-    # Reported upstream: https://github.com/patrick-kidger/equinox/issues/1186
-    "test_jax_transform_warn"
-
-    # Flaky: AssertionError: Non-linear scaling detected
-    "test_speed_buffer_while"
   ];
 
   pythonImportsCheck = [ "equinox" ];

--- a/pkgs/development/python-modules/jax-cuda12-pjrt/default.nix
+++ b/pkgs/development/python-modules/jax-cuda12-pjrt/default.nix
@@ -50,8 +50,8 @@ buildPythonPackage (finalAttrs: {
       .${stdenv.hostPlatform.system};
     hash =
       {
-        x86_64-linux = "sha256-U2owUpInbFdF77un61dXaEnFp8dzmKOp5h/TG69RAvA=";
-        aarch64-linux = "sha256-VvSifl8ZypFMD0QCU5RpqpLQG/cTNqzQ7Y/dwgqRvI0=";
+        x86_64-linux = "sha256-MJUVRT9Zyq2Vv3bIvGScJLwOPRLQe68895K+CCq97js=";
+        aarch64-linux = "sha256-NqwxbFiKDgjrZPhoJwUmrco4VtS3CxN2C36bc76F0s8=";
       }
       .${stdenv.hostPlatform.system};
   };

--- a/pkgs/development/python-modules/jax-cuda12-plugin/default.nix
+++ b/pkgs/development/python-modules/jax-cuda12-plugin/default.nix
@@ -39,42 +39,42 @@ let
     "3.11-x86_64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_x86_64";
       dist = "cp311";
-      hash = "sha256-1Vd82Ge9kmd2nkU7rYUNSAeoQ5a8l29jKlFe29d+SEs=";
+      hash = "sha256-lXMXyv8eawBqE1UXM3cN8jWidHBkviawJ0YP46WPWO4=";
     };
     "3.11-aarch64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_aarch64";
       dist = "cp311";
-      hash = "sha256-s5VfN10XkC8NJ+cFlnLNGWOlU0WVOkJpnk4HjOxyWtw=";
+      hash = "sha256-GkpKVqgyrTDgHBQIWx+iRApV+kzggI5AXnNlDv+K+zE=";
     };
     "3.12-x86_64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_x86_64";
       dist = "cp312";
-      hash = "sha256-iKVZCNd1sG3akqjE9MAVd44lulw2BbV/hLAAUvZujvE=";
+      hash = "sha256-oWugDXI2YUWCfKOajISfn+4Rd9ZzjJ7f1ys6n+31zxg=";
     };
     "3.12-aarch64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_aarch64";
       dist = "cp312";
-      hash = "sha256-sozPBbzAvHzLy9Mm2AKEZXTPbaA5FY52FHvZb1xvEYk=";
+      hash = "sha256-6vGmGVqg60i0IMcfIrktGXeNlIAkl3yuRkEnnV9sH4E=";
     };
     "3.13-x86_64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_x86_64";
       dist = "cp313";
-      hash = "sha256-vX3+0Xv6nQ4wFvjCpnZ8dHnZHhvf33kW6ysHQ1zEZY4=";
+      hash = "sha256-7QdNN5Z7zmHBuv3fndCoMCMGwS+QcAylrfo9CP3Uruc=";
     };
     "3.13-aarch64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_aarch64";
       dist = "cp313";
-      hash = "sha256-uaJwhdiTzFnCsoaxeJdV+Rzz6rHeobW+nmMvTJc5og4=";
+      hash = "sha256-dHnmfZ2Y1KD1JflPi4GIDcwFhjH23hGLkd6R2xtAle0=";
     };
     "3.14-x86_64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_x86_64";
       dist = "cp314";
-      hash = "sha256-U1F0LA/LIdqeCUoZZasg/eUlhih392kYpJCxtWZk1To=";
+      hash = "sha256-96ahiBNJ3DnyRIGioBn/KUuUkJuAqIge/iHUW2pfaR0=";
     };
     "3.14-aarch64-linux" = getSrcFromPypi {
       platform = "manylinux_2_27_aarch64";
       dist = "cp314";
-      hash = "sha256-MyEmmeG7sb7V0q4Urp/3Kh7tLQkqUearzAJ4prK4KHQ=";
+      hash = "sha256-iIb8V5df7gG+FyX4mLaSkvYVtwgwrvXC+95jgzNha10=";
     };
   };
 in

--- a/pkgs/development/python-modules/jax/default.nix
+++ b/pkgs/development/python-modules/jax/default.nix
@@ -6,7 +6,6 @@
   lapack,
   buildPythonPackage,
   fetchFromGitHub,
-  fetchpatch2,
   cudaSupport ? config.cudaSupport,
 
   # build-system
@@ -41,24 +40,17 @@ let
 in
 buildPythonPackage (finalAttrs: {
   pname = "jax";
-  version = "0.9.2";
+  version = "0.10.0";
   pyproject = true;
+  __structuredAttrs = true;
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "jax";
     # google/jax contains tags for jax and jaxlib. Only use jax tags!
     tag = "jax-v${finalAttrs.version}";
-    hash = "sha256-/vLCTAF46M1H0Q64RLM7+IFMofmjZmZ4IFzvm/y7zkg=";
+    hash = "sha256-/RCihrjONN/+QwyQRNEmlIa7JsCLzz+SkBe5sd+ThgU=";
   };
-
-  patches = [
-    # https://github.com/jax-ml/jax/pull/32840
-    (fetchpatch2 {
-      url = "https://github.com/Prince213/jax/commit/af5c211d49f3b99447db2252d2cc2b8e0fb54d1c.patch?full_index=1";
-      hash = "sha256-ijEd+MDe91qyYfE+aMzR5rNmTeGadin6Io8PIfJWc3o=";
-    })
-  ];
 
   build-system = [ setuptools ];
 

--- a/pkgs/development/python-modules/jaxlib/bin.nix
+++ b/pkgs/development/python-modules/jaxlib/bin.nix
@@ -18,7 +18,7 @@
 }:
 
 let
-  version = "0.9.2";
+  version = "0.10.0";
   inherit (python) pythonVersion;
 
   # As of 2023-06-06, google/jax upstream is no longer publishing CPU-only wheels to their GCS bucket. Instead the
@@ -49,65 +49,65 @@ let
       "3.11-x86_64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_x86_64";
         dist = "cp311";
-        hash = "sha256-msmVtLoaru2uDWnzGZh9UV3K7NRQW2QrYxL54VQ5NR8=";
+        hash = "sha256-m+IpmTpB5bK4TyNOzBml3gLzXt2xGVzwJ71TnhYB4V0=";
       };
       "3.11-aarch64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_aarch64";
         dist = "cp311";
-        hash = "sha256-MG3lSh3nOGyAbHI+NWzjMtkj73SPinLWdP77dIEh1Nw=";
+        hash = "sha256-PblOvIWTddlV3jUEGCrdfOFzPOPTDBXg7wMWAstRpVk=";
       };
       "3.11-aarch64-darwin" = getSrcFromPypi {
         platform = "macosx_11_0_arm64";
         dist = "cp311";
-        hash = "sha256-eF8XfD63jLfceXxV7VxLYxIUGEXJpoaVfkhLrL/OXog=";
+        hash = "sha256-J3Ay6fB0w/1f/R4MsD1P5m4nLeRyZnzbxBitmbIbZGo=";
       };
 
       "3.12-x86_64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_x86_64";
         dist = "cp312";
-        hash = "sha256-iLJ2px9PIHGx/S6SKr/WfIfGl3pVGhA2/rzqeNXvfiI=";
+        hash = "sha256-sL+4ZaB98ubXQYwLDCkt0pS1UAUjsd1YcrGA2yqkgNQ=";
       };
       "3.12-aarch64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_aarch64";
         dist = "cp312";
-        hash = "sha256-/vAthGhjtybnJFKZOIOoWW6sMl8ioux+qSHaD7xVCbQ=";
+        hash = "sha256-qh1w8aTifrQDZU5x4vso1XhtPpt3/BhH6MU4mICSfKQ=";
       };
       "3.12-aarch64-darwin" = getSrcFromPypi {
         platform = "macosx_11_0_arm64";
         dist = "cp312";
-        hash = "sha256-l8L75Yy+5KJ9lMpzXXCdIxspmrbtizsQdfUthk39MsE=";
+        hash = "sha256-fB2bRjMnx6IzPyEBFOywTyj+/FG6gjOoWiKAzOdb20I=";
       };
 
       "3.13-x86_64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_x86_64";
         dist = "cp313";
-        hash = "sha256-ttUAPjrdXDRqNK6e3EcFjLwttgyO1cUAllIhdtrwHJ8=";
+        hash = "sha256-0wPcMbZei3k9VgD4GxWDvgPcm4dqTBCz4lm2YJocvjs=";
       };
       "3.13-aarch64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_aarch64";
         dist = "cp313";
-        hash = "sha256-vvYe7zbtOM7BBp6pc/iK+eAzNeiE9lAew/5/YiKhVVs=";
+        hash = "sha256-bY14twcLNOTFu6X34Qkn5/Sqybab4X6bCliYVTpDOPM=";
       };
       "3.13-aarch64-darwin" = getSrcFromPypi {
         platform = "macosx_11_0_arm64";
         dist = "cp313";
-        hash = "sha256-UqADJQj4z1eRx6e+4UJTHucGw8BVGBF/sLbujV4X/ec=";
+        hash = "sha256-OEY1//VYmaKVu8gu5sb3c6MA54fcRyypK755q/qsg2k=";
       };
 
       "3.14-x86_64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_x86_64";
         dist = "cp314";
-        hash = "sha256-PXFRFApJNvMhiy0bE0PdI3vShlz1FEKIS22C/ohKPec=";
+        hash = "sha256-KkLPBMD4i8A7FQoX+n3bsvQOCWZn7IobhA7YeRPm5zU=";
       };
       "3.14-aarch64-linux" = getSrcFromPypi {
         platform = "manylinux_2_27_aarch64";
         dist = "cp314";
-        hash = "sha256-vkYnxC1Erdf+F9KE71ef+NFZ48tpR/ZDd1jzQXfoeOY=";
+        hash = "sha256-rUfgckMJeewhY3qkh9TcRkAouOm+JyaPN95pU2x240E=";
       };
       "3.14-aarch64-darwin" = getSrcFromPypi {
         platform = "macosx_11_0_arm64";
         dist = "cp314";
-        hash = "sha256-urFo0lVVRkRhvQdzI0hPaQxHHmnOiww5o5+4Gz46i/A=";
+        hash = "sha256-mLJmcpQ2cnQoc/ZbwDIWgZ/FUyXJnxRlkNAHwBcr/zA=";
       };
     };
 in
@@ -115,6 +115,8 @@ buildPythonPackage {
   pname = "jaxlib";
   inherit version;
   format = "wheel";
+
+  __structuredAttrs = true;
 
   # See https://discourse.nixos.org/t/ofborg-does-not-respect-meta-platforms/27019/6.
   src = (

